typescript/.changeset/quick-ravens-act.md

```diff
@@ -0,0 +1,9 @@
+---
+"@coinbase/agentkit-model-context-protocol": minor
+"@coinbase/agentkit-vercel-ai-sdk": minor
+"@coinbase/agentkit-langchain": minor
+"create-onchain-agent": minor
+"@coinbase/agentkit": minor
+---
+
+Migrate from npm to pnpm for dependency management
```

Comment:
Should this change be visible to end users?

Reply:
hmm, good point?

I was thinking it should go in the logs so contributors are aware of the change. If they continue to use npm, it will "work", but will run into issues once they make their PR and get blocked by the linters pnpm-lock.yml checks.

But you're right, the average user is not a contributor. To the average user, this does not deserve to be in the changelog

Reply:
I would omit from the changelog TBH, no need to bump the packages for this IMO, but we should notify devs with open PRs that they will have to switch to `pnpm`. cc @John-peterson-coinbase